server/migrations/20191024120045-0.1.0.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-param-reassign */
+const { runAsyncTasks } = require('../common/utils');
+
+const up = async db => {
+  const addDeleteTokenData = async () =>
+    db.collection('users').updateMany(
+      {
+        deleteToken: { $exists: false },
```

Comment:
it's not necessary to add these fields to all users. Refresh and confirmation tokens are added when they are needed. We add these fields when user decides to remove account. Why litter the document with redundant data?

Reply:
so is the migration needed in this case or not?